lib/modules/manager/dockerfile/extract.spec.ts

```diff
@@ -219,13 +219,13 @@ describe('modules/manager/dockerfile/extract', () => {
       const res = extractPackageFile('FROM quay.io:1234/node\n').deps;
       expect(res[0]).toMatchInlineSnapshot(`
         Object {
-          "autoReplaceStringTemplate": "{{lookupName}}{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}",
+          "autoReplaceStringTemplate": "{{packageName}}{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}",
```

Comment:
We probably need to migrate those too 🤔 As users can references `lookupName` in any templates.